src/lsm/groove.zig

```diff
@@ -1288,6 +1300,13 @@ pub fn GrooveType(
             };
         }
 
+        fn index_zeroes(comptime field: std.builtin.Type.StructField) bool {
+            for (groove_options.optional) |field_name| {
```

Comment:
Can we enforce comptime evaluation here with a `comptime for`?

Reply:
I think this is comptime evaluation already, since the only argument is comptime, but I made all invocations `comptime index_zeroes()` so that it is unambiguous.